src/sentry/static/sentry/app/views/organizationEventsV2/modalPagination.tsx

```diff
@@ -27,6 +37,7 @@ function buildTargets(event, location) {
   };
 
   const links = {};
```

Comment:
Can you declare this as `LinksType`

Reply:
Nah I can't. TS just complains about missing properties.

Reply:
I've added some type hints for the keys.